File: OnlineDeliveryServer/ClassLibrary/OrderItem.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClassLibrary
{
    public class OrderItem
    {
        public string Name { get; set; }
        public int Id { get; set; }
        public string Ingredients { get; set; }
        public double Price { get; set; }
        public int Quantity { get; set; }
        [ForeignKey("Order")]
        public int OrderId { get; set; }
        public Order Order { get; set; }
        public OrderItem() { }

        public OrderItem(string name, int id, string ingredients, double price)
        {
            Name = name;
            Id = id;
            Ingredients = ingredients;
            Price = price;
        }
    }
}

File: WebProject/src/model/OrderItem.ts
export class OrderItem
{
    id: number = 0;
  name: String = '';
  ingredients: String = '';
  price: number = 0;
  quantity: number =0;
  public ToString(orderItem: OrderItem) {
    return (
      'ID: ' +
      orderItem.id +
      'Name: ' +
      orderItem.name +
      'Quantity: '+
      orderItem.quantity+
      'Price: ' +
      orderItem.price
    );
  }
}